dev/update_ae.R

```diff
@@ -18,7 +18,8 @@ ae$AELAT <- if_else(ae$AESOC == "EYE DISORDERS",
   apply(ae, 1, function(x) sample(lat, 1)),
   NA_character_
 )
-admiral_ae <- ae
+admiral_ae <- ae %>%
+  add_labels(AELAT = "Laterality")
```

Comment:
You should do a `library(metatools)` for posterity :)

Reply:
DONE.